lib/p2p/Peer.ts

```diff
@@ -69,6 +61,14 @@ class Peer extends EventEmitter {
     return peer;
   }
 
+  public getStatus(): string {
```

Comment:
Is there a reason why you switched this from a getter method? Mainly just curious. I do think we should make this an arrow function tho to match the rest of the class.

Reply:
I switched it away from a getter syntax just as a style preference. I find this syntax more suitable for something like a pseudo property, or just a wrapper that you want it to look like a property. But `status` looked to me like something that might eventually need to retrieve/calculate, so I didn't find that syntax appropriate.